kuma/humans/fixtures/contributors.json

```diff
@@ -1,154 +1,170 @@
 [
```

Comment:
This file has been updated with the latest response from https://api.github.com/repos/mozilla/kuma/contributors, limited to the top 8 contributors for brevity sake.